packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx

```diff
@@ -528,6 +535,12 @@ const CreateServiceWizard: React.FC<Props> = ({
           trackWizardPageEvent={trackWizardPageEvent}
           {...serviceNextStepStatus}
         />
+        <button
+          type="submit"
+          disabled={continueButtonClicked || loadingCreateService}
+        >
+          Continue
+        </button>
```

Comment:
Isn't this button duplicate? 
what will happen with the button is currently shown from the WizardFooter?
Please add a screenshot or a video of the solution before we continue the review.

Reply:
yep it creates a duplicate at the WizardFooter, I guess I'll have to make changes in the CreateServiceWizardFooter.tsx